pkg/controllers/securitypolicy_controller.go

```diff
@@ -110,7 +120,7 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		if controllerutil.ContainsFinalizer(obj, util.FinalizerName) {
 			metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerDeleteTotal, METRIC_RES_TYPE)
 			if err := r.Service.DeleteSecurityPolicy(obj.UID); err != nil {
-				log.Error(err, "delete failed, would retry exponentially", "securitypolicy", req.NamespacedName)
+				log.Error(err, "deletion failed, would retry exponentially", "securitypolicy", req.NamespacedName)
```

Comment:
I feel "failed to delete"  would be more popular.

Reply:
ok